### ISSUE TITLE
[Jira] Recently Updated Issues fail with "JS heap out of memory" error

### ISSUE THREAD

**Issue by dannyc96** [raycast/extensions#7013]
### Extension

https://www.raycast.com/raycast/jira

#### Error:
```
Error: Worker terminated due to reaching memory limit: JS heap out of memory  NodeError:errors:399:5 [kOnExit]:worker:287:26 Worker.<computed>.onexit:worker:209:20[Jira] ...
```

### Steps To Reproduce

Go to Recently Updated Issues

**Comment by thomaslombart** [comment]
This may be due to an internal Raycast bug. We already have a fix internally, but we want to test it out a bit before releasing it. Thank you for your patience!

**Comment by thomaslombart** [comment]
Hey @dannyc96, we've fixed a performance issue in the 1.54.0 release that should make this type of bugs less likely to appear. Do you still have the error?

**Comment by thomaslombart** [comment]
I'll mark it as done as the error disappears on other extensions (such as Todoist). Feel free to re-open it if the issue persists for versions 1.54+.
